### PR TITLE
[WIP] Add Readme with permissions table

### PR DESCRIPTION
**Pull request by fabriziovigevani**
Added brief overview and permissions table.

TODO: 

- [ ] Add brief explanation of the main functions called  to create a Dandelion Org

- [ ] Add YAML spec

### CODE REVIEW

**Comment by yeqbfgxjiq** [comment]
@fabriziovigevani When you say "Add main functions to create template," do you mean add steps to run the template locally and/or deploy an actual DAO on Rinkeby or mainnet with the web interface or CLI?

**Review by yeqbfgxjiq — APPROVED** [review]
Looks great! Thanks for putting this together. Added a few updates to make the README format closer to the other repos.

**Comment by fabriziovigevani** [comment]
> When you say "Add main functions to create template," do you mean add steps to run the template locally and/or deploy an actual DAO on Rinkeby or mainnet with the web interface or CLI?

I meant adding a brief explanation of the main contract functions that are called to create a Dandelion Organization (updated comment) but maybe that should be in a separate technical doc?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,68 @@
...
+## Permissions
+
+| App               | Permission             | Grantee                 | Manager          | ACL Oracle       |
+| ----------------- | ---------------------- | ----------------------- | ---------------- | ---------------- |
+| Kernel            | APP_MANAGER            | Dandelion Voting        | Dandelion Voting | None             |
+| ACL               | CREATE_PERMISSIONS     | Dandelion Voting        | Dandelion Voting | None             |
+| EVMScriptRegistry | REGISTRY_MANAGER       | Dandelion Voting        | Dandelion Voting | None             |
+| EVMScriptRegistry | REGISTRY_ADD_EXECUTOR  | Dandelion Voting        | Dandelion Voting | None             |
+| Dandelion Voting  | CREATE_VOTES           | Time Lock               | Dandelion Voting | None             |
+| Dandelion Voting  | MODIFY_QUORUM          | Voting                  | Dandelion Voting | None             |
```
Comment (lkngtn) [inline README.md]: The use of "Voting" rather than "Dandelion Voting" seems inconsistent here.

Comment (fabriziovigevani) [reply README.md]: Good catch

**Pull request merged**